valhalla/thor/trafficalgorithm.h

```diff
@@ -1,67 +0,0 @@
-#ifndef VALHALLA_THOR_TRAFFICALGORITHM_H_
```

Comment:
this was an old school POC we did for traffic influenced routing. it no longer serves a purpose so it gets the :axe: